extension/src/popup/basics/layout/View/index.tsx

```diff
@@ -243,6 +248,7 @@ export const ViewInset: React.FC<ViewInsetProps> = ({
       hasTopBorder ? "View__inset--top-border" : "",
       hasScrollShadow ? "View__inset--scroll-shadows" : "",
       hasNoTopPadding ? "View__inset--no-top-padding" : "",
+      hasTopInput ? "View__inset--top-input" : "",
```

Comment:
this is a small styling fix for an issue I noticed: for screens where there is an input at the top of the screen (Manage assets, send payment), the input highlight was being cut off by the top header when the input was focused. This adds a small amount of space to around the input to accommodate for this highlight